internal/locale/locales/en-us.yaml

```diff
@@ -1145,23 +1083,25 @@ err_exec_recursion:
     To allow recursion, set [ACTIONABLE]{{.V1}}=true[/RESET]
 package_ingredient_alternatives:
   other: |
-    No results found for search term "[NOTICE]{{.V0}}[/RESET]". Did you mean:
+    No results found for search term {{.V0}}. Did you mean:
```

Comment:
It's still highlighted, but because it now takes multiple values it happens outside of the locale file:

https://github.com/ActiveState/cli/blob/02ba52766043b37a4fa9ebca874a7d8b9bf054ff/internal/runners/install/rationalize.go#L29-L29